docs/installation/k8s.md

```diff
@@ -0,0 +1,70 @@
+# Install on any Kubernetes Cluster
```

Comment:
Can we rename this to `custom.md` / `any.md` or something? All of the installs are technically k8s.

Reply:
done, I switched to `any.md`